.circleci/config.yml

```diff
@@ -63,15 +63,6 @@ commands:
     description: Commands to run for setup of every set of tests
     steps:
       - checkout
-      - run:
```

Comment:
These commands were not doing anything, we don't even use BitBucket. Removing them has had no effect on CI.